src/calendar/CalendarState.ts

```diff
@@ -29,13 +29,25 @@ import {
 import { CalendarProps } from "./index.d";
 import { useWeekStart } from "./useWeekStart";
 import { announce } from "../utils/LiveAnnouncer";
-import { generateDaysInMonthArray, isInvalid, useWeekDays } from "./__utils";
-
-export interface IUseCalendarProps extends CalendarProps {
+import { isInvalid, useWeekDays, generateDaysInMonthArray } from "./__utils";
+
+export type DateValue = string | number | Date;
+export interface CalendarStateInitialProps {
+  minValue?: DateValue;
+  maxValue?: DateValue;
+  isDisabled?: boolean;
+  isReadOnly?: boolean;
+  autoFocus?: boolean;
+  /** The current value (controlled). */
+  value?: DateValue;
+  /** The default value (uncontrolled). */
+  defaultValue?: DateValue;
+  /** Handler that is called when the value changes. */
+  onChange?: (value: DateValue) => void;
```

Comment:
Why we have changed it.
Any particular reason, We could have used the `CalendarProps` from `./index.d`

Reply:
Yup right, missed it sorry